avocado/core/varianter.py

```diff
@@ -362,7 +364,7 @@ def _process_default_params(self, args):
 
     def is_parsed(self):
         """
-        Reports whether the tree was already multiplexed
+        Reports whether the varianter was already parsed (stage1 vs. stage2)
```

Comment:
OK I removed the (stage1 vs. stage2) and I'll leave it on the reader to associate this with `parse`.